notebooks/regression/exoplanets.md

```diff
@@ -13,14 +13,14 @@ jupyter:
     name: mapie_local
 ---
 
-# Estimating the uncertainties of exoplanet masses
+# Estimating the uncertainties in the exoplanet masses
```

Comment:
Estimating the uncertainties in the exoplanet's mass ?

Reply:
DeepL gives me "exoplanet masses".

Reply:
OK, my mistake, let's trust deepl!